### ISSUE TITLE
Lots of ['None Found'] results for Japanese language.

### ISSUE THREAD

**Issue by wang-shuochen** [MaartenGr/KeyBERT#56]
So I was testing how this fares for the Japanese language. I used the bert tokenizer from the bert model (https://huggingface.co/cl-tohoku/bert-base-japanese-whole-word-masking). I could change the tokenizer to Sudachi or Meccab but I think the results will be more or less the same. 
```
from keybert import KeyBERT
# 日本語の事前学習モデル
MODEL_NAME = 'cl-tohoku/bert-base-japanese-whole-word-masking'
tokenizer = BertJapaneseTokenizer.from_pretrained(MODEL_NAME)

kw_model = KeyBERT()
kw_model.extract_keywords(tokenizer.tokenize("感染力の強い変異株の影響により、これまでに経験のないスピードで感染が拡大しています。"), keyphrase_ngram_range=(1, 2), stop_words=None)
```
The output I get is 
```
[[('感染', 1.0)],
 ['None Found'],
 ['None Found'],
 [('強い', 1.0)],
 [('変異', 1.0)],
 ['None Found'],
 ['None Found'],
 [('影響', 1.0)],
 [('により', 1.0)],
 ['None Found'],
 [('これ', 1.0)],
 [('まで', 1.0)],
 ['None Found'],
 [('経験', 1.0)],
 ['None Found'],
 [('ない', 1.0)],
 [('スピード', 1.0)],
 ['None Found'],
 [('感染', 1.0)],
 ['None Found'],
 [('拡大', 1.0)],
 ['None Found'],
 ['None Found'],
 ['None Found'],
 [('ます', 1.0)],
 ['None Found']]
```
It seems that the score is either 0 or 1 and the extraction does not seem very satisfactory. Any idea to improve the performance?

**Comment by MaartenGr** [comment]
I would advise you to use [`jieba`](https://github.com/fxsjy/jieba) for tokenization:

```python
from sklearn.feature_extraction.text import CountVectorizer
import jieba

def tokenize_zh(text):
    words = jieba.lcut(text)
    return words

vectorizer = CountVectorizer(tokenizer=tokenize_zh)
```

Then, simply pass the vectorizer to your KeyBERT instance:

```python
from keybert import KeyBERT

kw_model = KeyBERT(model="paraphrase-multilingual-mpnet-base-v2")
keywords = kw_model.extract_keywords(doc, vectorizer=vectorizer)
```

Do note that it would be best choosing an embedding model that best suits the language of your documents. In this case, I choose `paraphrase-multilingual-mpnet-base-v2` but perhaps other models are better.

**Comment by akash-cis** [comment]
> Thanks for the suggestion. "jieba" is targeted for the Chinese language I believe. For the Japanese vectorizer paraphrase-multilingual-mpnet-base-v2 does work better, with occasionally some not found error exists.

Can you please give your code link for the same as want to know where to pass the language variable that contains the list of other country language in which we want to extract the keyword
